pkg/netconf/configurator.go

```diff
@@ -8,24 +8,27 @@ import (
 
 	"github.com/metal-stack/metal-networker/pkg/exec"
 	"github.com/metal-stack/metal-networker/pkg/net"
+	"go.uber.org/zap"
 )
 
 // BareMetalType defines the type of configuration to apply.
 type BareMetalType int
 
 const (
-	// FileModeSystemd represents a file mode that allows systemd to read e.g. /etc/systemd/network files.
-	FileModeSystemd = 0644
-	// FileModeSixFourFour represents file mode 0644
-	FileModeSixFourFour = 0644
-	// FileModeDefault represents the default file mode sufficient e.g. to /etc/network/interfaces or /etc/frr.conf.
-	FileModeDefault = 0600
 	// Firewall defines the bare metal server to function as firewall.
 	Firewall BareMetalType = iota
 	// Machine defines the bare metal server to function as machine.
 	Machine
-	// SystemdUnitPath is the path where systemd units will be generated.
-	SystemdUnitPath = "/etc/systemd/system/"
+)
+const (
+	// fileModeSystemd represents a file mode that allows systemd to read e.g. /etc/systemd/network files.
+	fileModeSystemd = 0644
+	// fileModeSixFourFour represents file mode 0644
+	fileModeSixFourFour = 0644
```

Comment:
Do we need 2 different constants for the same permission?

Reply:
lets do these refactorings in a next PR.